tox.ini

```diff
@@ -48,6 +48,7 @@ commands =
 [testenv:docs,docs-py{38,39,310,311}]
 description = Build documentation
 extras = docs
+passenv = *
```

Comment:
This line ensures that set environment variables are just passed. Tested this on my fork, and it works.